src/app/features/users-search/users-search-container.component.ts

```diff
@@ -1,20 +1,25 @@
 import { Component } from '@angular/core';
-import { GithubUser } from '@models';
-import { select, Store } from '@ngrx/store';
+import { Store } from '@ngrx/store';
 import * as fromUsers from '@store/users-store';
+import { UsersSearchSelectorsService } from './users-search-selectors.service';
 
 @Component({
   selector: 'app-users-search-container',
   templateUrl: './users-search-container.component.html',
   styleUrls: ['./users-search-container.component.scss']
 })
 class UsersSearchContainerComponent {
-  users$ = this.store.pipe(select(fromUsers.selectAllUsers));
+  constructor(
```

Comment:
public user$:Observable<any[]> = this.service.user$;
public isLoading$:Observable<any[]> = this.service.isLoading$;

public show:Observable<boolean> = Observable.combinelatest(this.users$, this.isLoading$,(users, isLoading=>true));